### PR TITLE
Add Font Awesome integration

### PR DESCRIPTION
**Pull request by ghost**
**What**: Add Font Awesome integration

**Why**:

**How**:

**Checklist**:

- [x] Tests
- [x] Ready to be merged

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +20,9 @@
     "gatsby-plugin-sharp": "2.0.35",
     "gatsby-source-filesystem": "2.0.29",
     "gatsby-transformer-sharp": "2.1.18",
+    "i": "^0.3.6",
```
Comment (ri7nz) [inline package.json]: this is `i` for what @nagacoder  ?

Comment (nagacoder) [reply package.json]: oh my bad, i forget to remove it

File: package.json [hunk 2]
```diff
@@ -16,7 +20,9 @@
...
     "node-sass": "4.11.0",
+    "npm": "^6.9.0",
```
Comment (ri7nz) [inline package.json]: this is `npm` @nagacoder  ?

Comment (ri7nz) [reply package.json]: don't forget to update this PRs branch @nagacoder

**Review by zainfathoni — CHANGES_REQUESTED** [review]
Please [sign all your commits](https://github.com/devpendent/website/wiki/Git#git-commits-signing), otherwise, this PR will never be merged.
I've set the branch configuration to always require commit signing.

You can do it by `cherrypick`-ing your commits and signing the new commits.

**Pull request closed without merging**